migrations/dlu/1_security.sql

```diff
@@ -0,0 +1,18 @@
+-- root password is set by the docker image / env variables, so no need to change it here
```

Comment:
Should this exist as a standard migration to run no matter the configuration?

Reply:
Yes

Reply:
Surely this is just rerunning what people configure in mysql_secure_installation?

Reply:
Yeah, it is just a small change and not really 100% required. But it is recommend.

Reply:
AFAICT, the global_priv table is an addition by MariaDB and may break in the face of an (Oracle) MySQL database.

In general, I'm not a fan of adding more vendor specific code, that moves the codebase further away from the ability to support other DBs like Postgres.

Maybe put setup-specific migrations in a subfolder named `mariadb`?